content/cloud-computing/concepts/saas/saas.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: 'SaaS'
+Description: 'SaaS is a cloud computing service delivery model that makes software and applications available over the internet.'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
```

Comment:
Is this within the spirit of networking? It is accessed via the internet, but I think the connection is loose.

Reply:
I agree the association is loose, but I would vouch for this because isn't cloud computing a specialty within network engineering? This [CC blog post](https://www.codecademy.com/resources/blog/what-does-a-network-engineer-do/#:~:text=Cloud%20Networking%20Architect,with%20cloud%20services.) seems to reinforce this notion. Also, this may be beneficially for SEO-purposes/discoverability.